packages/backend/common/models/review.js

```diff
@@ -1,9 +1,19 @@
 const Joi = require('@hapi/joi')
+const {id} = require('./user') 
 
 const ReviewSchema = Joi.object({
-  id: Joi.string()
-    .alphanum()
-    .required()
+  id: id,
```

Comment:
`id` of ReviewSchema should not be using the same logic as the id of user.

Reply:
why is it? i thought all the id should have the logic

Reply:
In case the ids of reviews and users have different pattern, which we have not yet specified. But just to be future proof. 

It also doesn't make sense to use the id of a UserSchema to apply to ReviewSchema (although there is no different for now)

Reply:
Ok, makes sense to future proof